src/Toolchain/JobConfig.ts

```diff
@@ -27,7 +27,7 @@ class JobConfig extends JobCommand {
     super(cmd);
     this.jobType = JobType.tConfig;
     this.name = "config";
-    this.notiTitle = "Running onecc...";
+    this.notiTitle = "Running tools...";
```

Comment:
How about this:

You can edit `run` in `ToolchainEnv.ts` to apply notiTitle according to the toolchain name.

![image](https://github.com/Samsung/ONE-vscode/assets/17171963/8ddec9a1-7a90-4d92-801d-6c9ac82a4322)

Reply:
I know this is an awkward comment after answering you that 'Runnign tools' would do 😅

Reply:
It seems great to know which toolchain we are using. I will accept your advice.

However, I have another question about this PR, and I would like to hear your opinion.

1. Is it okay to keep notiTitle in JobConfig.ts as "Running tools..."? or revert it back to 'Running onecc...?
2. Should I change line 341 as shown below because we use the toolchain name in notifications of ToolchainEnv and ToolchainProvider?

Reply:
1. Sure. I think it's anyways okay to keep `notiTitle`. 
2. Yes, I think it's better to read. https://github.com/Samsung/ONE-vscode/pull/1698#discussion_r1342807624


Maybe @jyoungyun have another opinions, let's wait for that.